docs/upgradetest.md

```diff
@@ -0,0 +1,33 @@
+This guide is for Harbor upgrade test  
+=======
+
+# Before upgrade
+
+## Prepare data  
+1. At least three users, at least one has admin role.  
+2. Create at least one private and one public project.  
+3. Add several members to project, each role need one.  
+4. Login harbor as project admin, push at least 2 tag into project, include signed.  
+5. Login harbor as project developer, push at least 2 tag into project, include signed.  
+6. Repeate 4 and 5 to all projects.  
+7. Add at least one endpoint to harbor.  
+8. Add replication rule to each project, need 3 trigger type(all,not one project).  
+9. Add several system labels and tag at least one to a tag.  
+10. Add several project labels and tag at least one to a tag.  
+  
```

Comment:
At least these data are missing in populating and verification:

- Access log.
- Scan result of an image.
- Replication job log.
- Some system level configuration, such as Email, token expiration time (When you update the doc, be specific, DO NOT say, "update some system configuration" say "update Email smtp server to xxxx")